hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/InstructionToken.kt

```diff
@@ -455,3 +453,14 @@ internal fun tokenizeInstructions(
 
     return tokens.toLeft()
 }
+
+
+private fun TokenizerContext.Consumer.findMethodInsn(): MethodInsnNode? {
+    return find { node ->
+        when (node) {
+            is MethodInsnNode -> node
+            is LabelNode, is LineNumberNode -> null
+            else -> return null
```

Comment:
Unnecessary `return` here

Reply:
The return null will return the method, whereas the `-> null` will continue searching!

Reply:
Ah, missed it. Return semantics for inline functions is very cool, but sometimes a bit confusing 😅